api/src/chat/controllers/block.controller.spec.ts

```diff
@@ -20,6 +20,15 @@ import { LanguageRepository } from '@/i18n/repositories/language.repository';
 import { LanguageModel } from '@/i18n/schemas/language.schema';
 import { I18nService } from '@/i18n/services/i18n.service';
 import { LanguageService } from '@/i18n/services/language.service';
+import { LoggerService } from '@/logger/logger.service';
```

Comment:
```suggestion
```
Hi @MohamedAliBouhaouala, we can remove the LoggerService since it's automatically injected when we call buildTestingMocks function

![image](https://github.com/user-attachments/assets/c5065877-b04b-42b0-8296-fedd59a6313f)